invokeai/app/services/session_queue/session_queue_sqlite.py

```diff
@@ -356,6 +362,38 @@ def prune(self, queue_id: str) -> PruneResult:
             self.__lock.release()
         return PruneResult(deleted=count)
 
+    def prune_all(self) -> PruneResult:
```

Comment:
No need for a new method, can re-use the `clear` method - pass `DEFAULT_QUEUE_ID` for the arg